config/nim.cfg

```diff
@@ -44,7 +44,7 @@ path="$lib/core"
 path="$lib/pure"
 
 @if not windows:
-  nimblepath="/opt/nimble/pkgs/"
```

Comment:
This is a breaking change. Not sure how to support "/usr/share"

Reply:
This PR is updating all paths for [FHS](https://www.pathname.com/fhs/pub/fhs-2.3.html#PURPOSE18) compliance in one go, so it won't break it. Even today, if we pass /usr/bin to install.nimf, this configuration needs to be updated manually after installation.